index.d.ts

```diff
@@ -0,0 +1,19 @@
+export = Credstash;
+
+declare class Credstash {
+  list(options: Credstash.IOptions, callback: Credstash.Callback<string[]>): void;
+  list(callback: Credstash.Callback<string[]>): void;
+
+  get(value: string, options: Credstash.IOptions, callback: Credstash.Callback<string>): void;
+  get(value: string, callback: Credstash.Callback<string>): void;
```

Comment:
Currebtky the method blows up if options is not provided :/

Reply:
see chat on slack, it blew up for me too, but that's because I lost the context through util.promisify - the dreaded `this` was lost